src/app/[locale]/(home)/occasions/_actions/create-occasions.ts

```
@@ -0,0 +1,57 @@
+            revalidatePath("/occasions");
+        }
+
+        // Return structured API response
+        return {
+            ok: res.ok,
+            status: res.status,
+            payload: payload as T | ApiErrorResponse,
+        };
+    } catch (error) {
+        // Log error for debugging
+        console.error("Create occasion error:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)